### PR TITLE
Add failure reason note example text

### PR DESCRIPTION
**Pull request by David-Feetenby**
Add placeholder text to failure reason text boxes

### CODE REVIEW

**Review by thomasleese — COMMENTED** [review]
Changes look good, could we take the opportunity to move the strings from this file in to the [`helpers.en.yml` locale file](https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/config/locales/helpers.en.yml)?

Here's an example of where we've done that for other forms: https://github.com/DFE-Digital/apply-for-qualified-teacher-status/commit/e74a05f9d98086e1bf41dbce75c4d89466869c36

File: app/views/assessor_interface/assessment_sections/show.html.erb [hunk 1]
```diff
@@ -72,7 +72,11 @@
       <%= f.govuk_check_boxes_fieldset :selected_failure_reasons, legend: { size: "s" } do %>
         <% @assessment_section_view_object.assessment_section.failure_reasons.each do |failure_reason| %>
           <%= f.govuk_check_box "#{failure_reason}_checked", true, label: { text: t(".failure_reasons.#{failure_reason}") }, link_errors: true do %>
-            <%= f.govuk_text_area "#{failure_reason}_notes", label: { text: "Note to the applicant", size: "s" }, hint: { text: "Use this space to tell the applicant what they need to do. Make sure your instructions are clear." } %>
+            <%= f.govuk_text_area "#{failure_reason}_notes",
+              label: { text: t("helpers.label.assessor_interface_assessment_section_form.failure_reason_notes"), size: "s" },
+              hint: { text: t("helpers.hint.assessor_interface_assessment_section_form.failure_reason_notes")},
+              placeholder: t("helpers.placeholder.assessor_interface_assessment_section_form.failure_reason_notes")
+              %>
```
Comment (thomasleese) [inline app/views/assessor_interface/assessment_sections/show.html.erb:79]: The indentation is a little off here, I'd expect it to look something more like:

```suggestion
            <%= f.govuk_text_area "#{failure_reason}_notes",
                                  label: { text: t("helpers.label.assessor_interface_assessment_section_form.failure_reason_notes"), size: "s" },
                                  hint: { text: t("helpers.hint.assessor_interface_assessment_section_form.failure_reason_notes") },
                                  placeholder: t("helpers.placeholder.assessor_interface_assessment_section_form.failure_reason_notes") %>
```

File: config/locales/helpers.en.yml [hunk 2]
```diff
@@ -36,6 +37,7 @@ en:
         age_range_min: From
         age_range_max: To
         age_range_note: If you've entered a new range please explain why (optional)
+        failure_reason_notes: "Note to the applicant"
```
Comment (thomasleese) [inline config/locales/helpers.en.yml:40]:
```suggestion
        failure_reason_notes: Note to the applicant
```

The quotes can mostly be avoided when using YAML.

File: config/locales/helpers.en.yml [hunk 3]
```diff
@@ -61,6 +63,9 @@ en:
         add_another_options:
           true: "Yes"
           false: "No"
+    placeholder:
+      assessor_interface_assessment_section_form:
+        failure_reason_notes: "Example: The right-hand section of your teaching qualification document is missing, please take a new image and upload it."
```
Comment (thomasleese) [inline config/locales/helpers.en.yml:68]:
```suggestion
        failure_reason_notes: Example: The right-hand section of your teaching qualification document is missing, please take a new image and upload it.
```

**Review by thomasleese — COMMENTED** [review]
Looks good, I've just got a few style comments.

File: config/locales/helpers.en.yml (hunk 3 above)
Comment (David-Feetenby) [reply config/locales/helpers.en.yml:68]: Do i not need the speech marks here because of 'example:'?

**Review by thomasleese — APPROVED** [review]

**Pull request merged**